get_download_info_tester.py

```diff
@@ -0,0 +1,12 @@
+from pymatgen.ext.matproj import MPRester, TaskType
+import os
+material_ids = ["mp-32800", "mp-23494"]
+task_types = [TaskType.GGA_OPT, TaskType.GGA_UNIFORM]
+file_patterns = ["vasprun*", "OUTCAR*"]
+with MPRester(os.environ["MP_API_KEY"]) as mpr:
+    meta, urls = mpr.get_download_info(
+        material_ids, task_types=task_types, file_patterns=file_patterns
+    )
+
+print(meta)
```

Comment:
Some stray `print` statements here. Was this file meant to be committed?

Reply:
Yes, this was meant to be an example of how to use the MPRester's `get_download_info` function. i can remove it if we intend to include it somewhere else.

Reply:
This would be better included in documentation somewhere, not as a script at the root level